sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -263,6 +263,7 @@ RT::PiProgram ProgramManager::loadProgram(OSModuleHandle M,
               << getRawSyclObjImpl(Context) << ")\n";
   }
 
+  const RT::PiContext &Ctx = getRawSyclObjImpl(Context)->getHandleRef();
```

Comment:
Suggest moving declaration of this variable closer to use site.

Reply:
I'd love to, but unfortunately, there's no clear way to do this - there is a second use way below in the main function scope.

Reply:
Should I consider this resolved?